gateway-service/src/test/java/org/zowe/apiml/acceptance/config/GatewayOverrideConfig.java

```diff
@@ -78,7 +78,6 @@ public SimpleRouteLocator simpleRouteLocator() {
 
     @Bean
     public ApplicationRegistry registry() {
-
```

Comment:
I would revert changes like this, it is more readable then

Reply:
do you mean this empty space?

Reply:
yes, I am fine if you leave them here, it is just increasing the changes and (probably) not adding value